app/templates/events/view.hbs

```diff
@@ -1,6 +1,59 @@
 {{#unless (includes session.currentRouteName 'events.view.edit')}}
-  {{!-- Tabs for the event view page will come here --}}
-  {{outlet}}
+  <div class="ui grid stackable">
+    <div class="row">
+      <div class="four wide column">
+        <h2 class="ui header">Fossasia Test</h2>
+      </div>
+      <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
+        {{#if (or device.isMobile )}}
```

Comment:
You can remove the or here like I said. 
It can just be `{{#if device.isMobile}}`